features/admin.feature

```diff
@@ -222,6 +225,14 @@ Feature:  Administration
     And I should not see "Delete"
     And I should see "search.gov"
 
+    When I follow "Sitemaps"
+    Then I should see "search.gov/sitemap.xml"
+    And I follow "Delete" and confirm "Are you sure you want to delete this sitemap?"
+    Then I follow "Create Sitemap"
```

Comment:
The `Then` line I mentioned above concludes the deletion test. Because you're now starting a creation test, I would change `Then` to `When`:
`When I follow "Create New"`

(Note that you will need to use `Create New` instead of `Create Sitemap`, as we're not using a custom create label.

Reply:
The reason I had originally changed the label to "Create Sitemap" instead of going with the default "Create New" was because when running this scenario, which tests that new `searchgov_domains` can be created and that new `sitemaps` can be created, cucumber was giving an "ambiguity error" because there were two lines that said "When I follow 'Create New'".  Is there another way around this then changing the label of the button to create sitemaps, besides testing that feature in a different scenario?